plexapi/myplex.py

```diff
@@ -682,6 +683,13 @@ def tidal(self):
         req = requests.get(self.MUSIC + 'hubs/', headers={'X-Plex-Token': self._token})
         elem = ElementTree.fromstring(req.text)
         return self.findItems(elem)
+      
+   def iptv(self):
+        """ Returns a list of IPTV Hub items :class:`~plexapi.library.Hub`
+        """
+        req = requests.get(self.IPTV + 'hubs/sections/all/', headers={'X-Plex-Token': self._token})
```

Comment:
Same question regarding the session here.

Reply:
Doesn't look like there's a stored Session here for me to use. Was just mimicking the other functions that already exist, which all call request.get()